cmd/download.go

```diff
@@ -316,9 +319,10 @@ func download(folder, release, url string,
 			}
 		}
 	}
-
-	for _, image := range images {
-		fmt.Fprintf(os.Stdout, "Processing artifact %s\n", image.Artifact)
+	var currentImage int = 1
+	for i, image := range images {
+		var imagesLength int = len(images) - 1
+		fmt.Fprintf(os.Stdout, "Processing artifact [%d/%d]: %s\n", i, imagesLength, image.Artifact)
```

Comment:
The "i" here should be currentImage instead, although since this is the only place it's used, I'd just drop the currentImage var declaration altogether and use i+1 in the printf.
Similarly, I'd drop the imagesLength and just use len(images) in the printf (which shouldn't be decremented here).
In my case, for example, I have 174 images in my mapping.txt, but the progress indications are going from 0/173 to 173/173, instead of 1/174 to 174/174.